apps/bfd-pipeline/bfd-pipeline-shared-test-utils/pom.xml

```diff
@@ -31,6 +31,18 @@
             <artifactId>bfd-pipeline-shared-utils</artifactId>
             <version>${project.version}</version>
         </dependency>
+        <dependency>
+            <groupId>gov.cms.bfd</groupId>
+            <artifactId>bfd-shared-utils</artifactId>
+            <version>${project.version}</version>
+        </dependency>
```

Comment:
When my open PR is merged, this should be able to be moved to test scope.

Reply:
Why is that?  The `PipelineTestUtils` class requires `DatabaseSchemaManager` to compile.

SInce `bfd-pipeline-shared-test-utils` is a test library it is fine for it to use production code in compile scope.  It's only the other way around that's problematic.

It turns out this dependency declaration is redundant in this pom so I'll remove it.

Reply:
Oops you're right I read it as the test version but this is the normal one. Ignore!